scripts/benchmark.js

```diff
@@ -46,10 +60,79 @@ const table = new Table({
   },
 });
 
-let results = [];
+let results = [],
+    code,
+	gzippedCode;
 
-const code = fs.readFileSync(filename, "utf8");
-const gzippedCode = zlib.gzipSync(code);
+function installPackage() {
+  const command = "npm install --prefix " + pathToScripts + " " + packagename;
+
+  try {
+    child.execSync(command);
+  }
+  catch(e) {
+	// Unecessary to print out error as the failure of the execSync will print it anyway
+    process.exit(1);
+  }
+}
+
+function uninstallPackage() {
+  const command = "npm uninstall --prefix " + pathToScripts + " " + packagename.split("@")[0];
+
+  try {
+    child.execSync(command);
+  }
+  catch(e) {
+    console.error('Error uninstalling package ' + packagename + ': ' + e);
+    process.exit(1);
+  }
+}
+
+function checkFile() {
+  // If filename has not been passed as an argument, attempt to resolve file from package.json
+  filename = filename ? path.join(pathToScripts, "node_modules", filename) : require.resolve(packagename.split("@")[0]);
+  console.log("file: " + path.basename(filename));
+
+  if(!filename || !pathExists(filename)) {
+    console.error("File not found. Exiting.");
+	!script.offline && uninstallPackage();
```

Comment:
The indenting looks fine when viewed in the actual script?  I will manually space it to see if that fixes it.